<launch>
	<arg name="veh"/>
	<arg name="config" default="baseline"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<arg name="pkg_name" default="fleet_planning" doc="name of the package"/>
	<arg name="node_name" default="actions_dispatcher_node" doc="name of the node"/>
	<arg name="map_name" default="autolab_tiles_map"/>
	<arg name="map_dir" default="$(find fleet_planning)/src/maps/"/>
	<arg name="target_node" default="240"/>
	<param name="veh" value="$(arg veh)"/>
	<param name="map_name" value="$(arg map_name)"/>
	<param name="map_dir" value="$(arg map_dir)"/>
	<param name="target_node" value="$(arg target_node)"/>

	<group ns="$(arg veh)">
	    <node  pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	        <rosparam command="load" file="$(find duckietown)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/>
	    </node>
	</group>

	<!-- Publication -->
	<!-- ~topic: msg_type. Description. -->

	<!-- Subscription -->
	<!-- ~topic: msg_type. Description. -->

</launch>
